bundles/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/protocol/xml/XMLConnection.java

```diff
@@ -74,7 +74,7 @@ public XMLConnection(String host) {
             connection.setRequestMethod("POST");
             connection.setRequestProperty("Content-Length", Integer.toString(message.length()));
 
-            connection.setConnectTimeout(5); // set a timeout in case the device is not reachable (went offline)
+            connection.setConnectTimeout(5000); // set a timeout in case the device is not reachable (went offline)
```

Comment:
Should this be constant, that could be a good place to include the unit in the name, because I hope that is milliseconds?

Reply:
Yes, that is milliseconds. This is the crux of the fix. Before I thought these were seconds (wanted 5 seconds).

Sure, a constant could be done added.